src/main/java/com/faforever/client/main/MainController.java

```diff
@@ -456,4 +459,31 @@ public void onNavigateEvent(NavigateEvent navigateEvent) {
   private AbstractViewController<?> loadView(NavigationItem item) {
     return noCatch(() -> viewCache.get(item, () -> uiService.loadFxml(item.getFxmlFile())));
   }
+
+  public void onRevealMapFolder() {
+    Path mapPath = preferencesService.getPreferences().getForgedAlliance().getCustomMapsDirectory();
+    try {
+      FileUtils.openFolder(mapPath);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public void onRevealModFolder() {
+    Path mapPath = preferencesService.getPreferences().getForgedAlliance().getModsDirectory();
+    try {
+      FileUtils.openFolder(mapPath);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
+  }
+
+  public void onRevealLogFolder() {
+    Path mapPath = preferencesService.getFafLogDirectory();
+    try {
+      FileUtils.openFolder(mapPath);
+    } catch (IOException e) {
+      e.printStackTrace();
+    }
```

Comment:
Don't do `e.printStackTrace()`, [as Codacy reports](https://www.codacy.com/app/FAForever/downlords-faf-client/pullRequest?prid=629971). But once you use the platform service there won't be an `IOException` anyway.